components/next_steps_view/steps/invite_members_step/invite_members_step.tsx

```diff
@@ -277,6 +278,20 @@ class InviteMembersStep extends React.PureComponent<Props, State> {
 
     render(): JSX.Element {
         const linkBtn = this.props.isAdmin ? <UpgradeLink telemetryInfo='click_upgrade_invite_members_step'/> : <NotifyLink/>;
+        let finishMessage = (
+            <FormattedMessage
+                id='next_steps_view.invite_members_step.finish'
+                defaultMessage='Finish'
+            />
+        );
+        if (this.props.downloadAppsAsNextStep) {
+            finishMessage = (
+                <FormattedMessage
+                    id='next_steps_view.invite_members_step.next_step'
+                    defaultMessage='Next step'
+                />
+            );
+        }
```

Comment:
Wondering here if there is a way to have this step know if it's the last one and not tied to the specific step we are adding right now. We will be experimenting and changing steps/order so it would be ideal if it was smarter here

Reply:
Doesn't need to be part of this PR specifically, just wondering if it's a possible future improvement

Reply:
Yeah, I was thinking about that, too. Its fairly doable; you could pass in a "isLastStep" flag [here](https://github.com/mattermost/mattermost-webapp/blob/master/components/next_steps_view/next_steps_view.tsx#L227) . I think the bigger issue that needs addressing to do that is to standardize the button wording in these components a little more. Most of them are specific to the step at hand.

An alternate approach would be to rename the action in invite_members_step and the new download_apps_step to also be specific to the step at hand.

Reply:
Let's create a ticket for this to see what we can do to improve this in the future

Reply:
Created as https://mattermost.atlassian.net/browse/MM-38643